src/renderers/shared/fiber/ReactChildFiber.js

```diff
@@ -72,6 +72,9 @@ const {
   Deletion,
 } = ReactTypeOfSideEffect;
 
+const internalErrorMessage =
+  'This error is likely caused by a bug in React. Please file an issue.';
```

Comment:
Hrrm, didn't think about that. Don't think it makes a substantial difference but we can change later.